OktaLogger.podspec

```diff
@@ -54,6 +55,14 @@ Pod::Spec.new do |s|
         'Sources/OktaLogger/AppCenterLogger',
         'Sources/OktaLogger/InstabugLogger'
       ]
+      core.ios.dependency 'OktaLogger/Analytics'
+  end
+
+  s.subspec 'Analytics' do |analytics|
+      analytics.ios.source_files = [
+        'Sources/OktaLogger/Analytics/*'
+      ]
+      analytics.ios.dependency 'AppCenter', '4.3.0'
```

Comment:
FYI, this pod has a bad constraint, it can be built as static lib only, so that constraint will affect the whole logger pod. I suggest to create separate spec - OktaLoggerAnalytics that will be built on top of Core and depend on AppCenter

Reply:
How we can use `OktaLoggerAnalytics` with core because we are trying to add Analytics part of OktaLogger core?

Reply:
That is my main question, should Analytics be really part of logger core? Ideally core should not have any dependencies on third party libs and has just do simple logging into standard output

Reply:
Agree with this, Analytics should not be part of Logger/Core. PushSDK depends on Logger/Core. Why would 3rd party devs want an AppCenter pod in their dependencies when using the PushSDK?